### PR TITLE
doc: Add Akenza Core webhook integration guide

### PR DESCRIPTION
**Pull request by nejraselimovic**
#### Summary
Adding a guide for Webhook integration with Akenza Core.
Reference: https://github.com/TheThingsIndustries/lorawan-stack/issues/2115

#### Testing
Tested this integration with TTS running locally.

#### Checklist
- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md`. The target branch is set to `master` if the changes are fully compatible with existing API, database, configuration and CLI.
- [x] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [ ] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

**Comment by nejraselimovic** [comment]
@benolayinka @laurensslats just to remind you that I'm still waiting for a review on this

File: doc/content/integrations/webhooks/cloud-integrations/akenza-core/akenza-setup/_index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+title: "Akenza Core Setup"
+description: ""
+weight: 1
+---
+
+This section shows how to prepare Akenza Core setup before creating a Webhook integration on {{% tts %}}.
+
+<!--more-->
+
+By logging in to your Akenza Core user account and navigating to **QUICK START** tab in the **Environment**, you can see the order of steps to be taken to connect a device and manipulate the data coming from it.
```
Comment (benolayinka) [inline doc/content/integrations/webhooks/cloud-integrations/akenza-core/akenza-setup/_index.md:11]:
```suggestion
Log in to Akenza Core and navigate to the **Quick Start** tab in the **Environment** to see the steps to be taken to connect a device and manipulate the data coming from it. 
```

File: doc/content/integrations/webhooks/cloud-integrations/akenza-core/akenza-setup/_index.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+This section shows how to prepare Akenza Core setup before creating a Webhook integration on {{% tts %}}.
+
+<!--more-->
+
+By logging in to your Akenza Core user account and navigating to **QUICK START** tab in the **Environment**, you can see the order of steps to be taken to connect a device and manipulate the data coming from it. 
+
+{{< figure src="quick-start.png" alt="Steps to connect a device" >}}
+
+First step is setting up a domain, which can be done by selecting the **ADD DOMAIN** button in **Domains** submenu.
```
Comment (benolayinka) [inline doc/content/integrations/webhooks/cloud-integrations/akenza-core/akenza-setup/_index.md:15]:
```suggestion
Set up a Domain by selecting the **Add Domain** button in **Domains** submenu. 
```

File: doc/content/integrations/webhooks/cloud-integrations/akenza-core/akenza-setup/_index.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+<!--more-->
+
+By logging in to your Akenza Core user account and navigating to **QUICK START** tab in the **Environment**, you can see the order of steps to be taken to connect a device and manipulate the data coming from it. 
+
+{{< figure src="quick-start.png" alt="Steps to connect a device" >}}
+
+First step is setting up a domain, which can be done by selecting the **ADD DOMAIN** button in **Domains** submenu. 
+
+Give a **Name** to your domain and choose **HTTP** for **Technology** in order to implement a Webhook integration. **Domain Secret** is auto-generated, while **Uplink Function** and **Downlink Function** may retain **Passthrough** value.
```
Comment (benolayinka) [inline doc/content/integrations/webhooks/cloud-integrations/akenza-core/akenza-setup/_index.md:17]:
```suggestion
Give a **Name** to your domain and choose **HTTP** for **Technology** in order to implement a Webhook integration. The **Domain Secret** is auto-generated, while **Uplink Function** and **Downlink Function** may retain a **Passthrough** value. 
```

File: doc/content/integrations/webhooks/cloud-integrations/akenza-core/akenza-setup/_index.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+By logging in to your Akenza Core user account and navigating to **QUICK START** tab in the **Environment**, you can see the order of steps to be taken to connect a device and manipulate the data coming from it. 
+
+{{< figure src="quick-start.png" alt="Steps to connect a device" >}}
+
+First step is setting up a domain, which can be done by selecting the **ADD DOMAIN** button in **Domains** submenu. 
+
+Give a **Name** to your domain and choose **HTTP** for **Technology** in order to implement a Webhook integration. **Domain Secret** is auto-generated, while **Uplink Function** and **Downlink Function** may retain **Passthrough** value. 
+
+Select **SAVE** to finish.
```
Comment (benolayinka) [inline doc/content/integrations/webhooks/cloud-integrations/akenza-core/akenza-setup/_index.md:19]:
```suggestion
Select **Save** to finish.
```

File: doc/content/integrations/webhooks/cloud-integrations/akenza-core/akenza-setup/_index.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+{{< figure src="creating-domain.png" alt="Creating a new domain" >}}
+
+In a similar manner, you need to create a new device type in **Device Types** submenu, where you just need to select **ADD DEVICE TYPE** button, provide a **Name** and click **SAVE**.
```
Comment (benolayinka) [inline doc/content/integrations/webhooks/cloud-integrations/akenza-core/akenza-setup/_index.md:23]:
```suggestion
Next, create a new device type in the **Device Types** submenu. Select the **Add Device Type** button, provide a **Name** and click **Save**.
```

File: doc/content/integrations/webhooks/cloud-integrations/akenza-core/akenza-setup/_index.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+{{< figure src="creating-device-type.png" alt="Creating a device type" >}}
+
+When you have defined a domain and a device type, you can create a new device by selecting **ADD DEVICE** button in **Inventory** submenu.
```
Comment (benolayinka) [inline doc/content/integrations/webhooks/cloud-integrations/akenza-core/akenza-setup/_index.md:27]:
```suggestion
When you have defined a domain and a device type, you can create a new device by selecting the **Add device** button in the **Inventory** submenu. 
```

File: doc/content/integrations/webhooks/cloud-integrations/akenza-core/akenza-setup/_index.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+{{< figure src="creating-domain.png" alt="Creating a new domain" >}}
+
+In a similar manner, you need to create a new device type in **Device Types** submenu, where you just need to select **ADD DEVICE TYPE** button, provide a **Name** and click **SAVE**.
+
+{{< figure src="creating-device-type.png" alt="Creating a device type" >}}
+
+When you have defined a domain and a device type, you can create a new device by selecting **ADD DEVICE** button in **Inventory** submenu. 
+
+Provide a **Name** for your device, select previously created device type and domain, and do not forget to generate **Device ID**.
```
Comment (benolayinka) [inline doc/content/integrations/webhooks/cloud-integrations/akenza-core/akenza-setup/_index.md:29]:
```suggestion
Provide a **Name** for your device, select the previously created device type and domain, and do not forget to generate a **Device ID**.
```

File: doc/content/integrations/webhooks/cloud-integrations/akenza-core/akenza-setup/_index.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+{{< figure src="creating-device.png" alt="Creating a new device" >}}
+
+Once the device is created, select it in **Inventory** and scroll down to the bottom of the page. Under **Endpoints** you can find and copy **HTTP Uplink URL** that you need for the Webhook integration.
```
Comment (benolayinka) [inline doc/content/integrations/webhooks/cloud-integrations/akenza-core/akenza-setup/_index.md:33]:
```suggestion
Once the device is created, select it in **Inventory** and scroll down to the bottom of the page. Under **Endpoints**, you will find the **HTTP Uplink URL** that you need for the Webhook integration.
```

File: doc/content/integrations/webhooks/cloud-integrations/akenza-core/tts-setup/_index.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+
+Once you have prepared the setup in Akenza Core, follow this section to create the Webhook integration.
+
+<!--more-->
+
+>Note: this section follows the [HTTP Webhooks]({{< ref "/integrations/webhooks" >}}) guide. 
+
+Fill in the **Webhook ID** field and choose **JSON** for **Webhook format**. 
+
+Paste the copied **HTTP Uplink URL** from Akenza Core in **Base URL** field.
```
Comment (benolayinka) [inline doc/content/integrations/webhooks/cloud-integrations/akenza-core/tts-setup/_index.md:15]:
```suggestion
Paste the copied **HTTP Uplink URL** from Akenza Core in the **Base URL** field.
```

File: doc/content/integrations/webhooks/cloud-integrations/akenza-core/tts-setup/_index.md [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+
+In order to send uplink messages to Akenza Core, check the **Enabled** box next to the **Uplink message**.
```
Comment (benolayinka) [inline doc/content/integrations/webhooks/cloud-integrations/akenza-core/tts-setup/_index.md:17]:
```suggestion
To send uplink messages to Akenza Core, check the **Enabled** box next to **Uplink message**. 
```

File: doc/content/integrations/webhooks/cloud-integrations/akenza-core/tts-setup/_index.md [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+
+Fill in the **Webhook ID** field and choose **JSON** for **Webhook format**. 
+
+Paste the copied **HTTP Uplink URL** from Akenza Core in **Base URL** field.
+
+In order to send uplink messages to Akenza Core, check the **Enabled** box next to the **Uplink message**. 
+
+{{< figure src="creating-webhook.png" alt="Akenza Core webhook" >}}
+
+After creating the integration, you will be able to see uplink messages in JSON format coming to Akenza Core if you navigate to **Inventory** &#8594; device &#8594; **DATA**.
```
Comment (benolayinka) [inline doc/content/integrations/webhooks/cloud-integrations/akenza-core/tts-setup/_index.md:21]:
```suggestion
After creating the integration, you will be able to see uplink messages in JSON format in Akenza Core in the **Inventory** &#8594; **Device** &#8594; **Data** menu.
```

File: doc/content/integrations/webhooks/cloud-integrations/akenza-core/akenza-setup/_index.md [hunk 12]
```diff
@@ -8,26 +8,26 @@ This section shows how to prepare Akenza Core setup before creating a Webhook in
 
 <!--more-->
 
-By logging in to your Akenza Core user account and navigating to **QUICK START** tab in the **Environment**, you can see the order of steps to be taken to connect a device and manipulate the data coming from it. 
+Log in to Akenza Core and navigate to **Quick Start** tab in the **Environment** to see the steps to be taken to connect a device and manipulate the data coming from it.
```
Comment (benolayinka) [inline doc/content/integrations/webhooks/cloud-integrations/akenza-core/akenza-setup/_index.md:11]:
```suggestion
Log in to Akenza Core and navigate to the **Quick Start** tab in the **Environment** to see the steps to be taken to connect a device and manipulate the data coming from it. 
```

File: doc/content/integrations/webhooks/cloud-integrations/akenza-core/akenza-setup/_index.md [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+
+Select **Save** to finish.
+
+{{< figure src="creating-domain.png" alt="Creating a new domain" >}}
+
+Next, create a new device type in the **Device Types** submenu. Select the **Add Device Type** button, provide a **Name** and click **Save**.
+
+{{< figure src="creating-device-type.png" alt="Creating a device type" >}}
+
+When you have defined a domain and a device type, you can create a new device by selecting **Add Device** button in the **Inventory** submenu.
```
Comment (laurensslats) [inline doc/content/integrations/webhooks/cloud-integrations/akenza-core/akenza-setup/_index.md:27]:
```suggestion
After defining the domain and a device type, you can create a new device by selecting **Add Device** button in the **Inventory** submenu. 
```

File: doc/content/integrations/webhooks/cloud-integrations/akenza-core/tts-setup/_index.md [hunk 14]
```diff
@@ -0,0 +1,21 @@
+---
+title: "Creating a Webhook"
+description: ""
+weight: 2
+---
+
+Once you have prepared the setup in Akenza Core, follow this section to create the Webhook integration.
```
Comment (laurensslats) [inline doc/content/integrations/webhooks/cloud-integrations/akenza-core/tts-setup/_index.md:7]:
```suggestion
Once you completed the setup in Akenza Core, follow this section to create the Webhook integration.
```

File: doc/content/integrations/webhooks/cloud-integrations/akenza-core/tts-setup/_index.md [hunk 15]
```diff
@@ -0,0 +1,21 @@
...
+
+Fill in the **Webhook ID** field and choose **JSON** for **Webhook format**. 
+
+Paste the copied **HTTP Uplink URL** from Akenza Core in the **Base URL** field.
+
+To send uplink messages to Akenza Core, check the **Enabled** box next to the **Uplink message**. 
+
+{{< figure src="creating-webhook.png" alt="Akenza Core webhook" >}}
+
+After creating the integration, you will be able to see uplink messages in JSON format in Akenza Core if you navigate to the **Data** tab of the created device (which can be found in the **Inventory**).
```
Comment (laurensslats) [inline doc/content/integrations/webhooks/cloud-integrations/akenza-core/tts-setup/_index.md:21]:
```suggestion
After creating the integration, you are able to see uplink messages in JSON format in Akenza Core in the **Data** tab of the created device (which can be found in the **Inventory**).
```

File: doc/content/integrations/webhooks/cloud-integrations/akenza-core/tts-setup/_index.md (hunk 14 above)
Comment (benolayinka) [reply doc/content/integrations/webhooks/cloud-integrations/akenza-core/tts-setup/_index.md]: @laurensslats you are creating more typos!

Once you have prepared or once you have completed

File: doc/content/integrations/webhooks/cloud-integrations/akenza-core/tts-setup/_index.md (hunk 15 above)
Comment (benolayinka) [reply doc/content/integrations/webhooks/cloud-integrations/akenza-core/tts-setup/_index.md]: Haha you have to ignore @laurensslats on this, the english is "You will be able to"

After creating the integration, you will be able to see uplink messages in JSON format in Akenza Core in the **Data** tab of the created device (which can be found in the **Inventory**).

File: doc/content/integrations/webhooks/cloud-integrations/akenza-core/akenza-setup/_index.md [hunk 16]
```diff
@@ -0,0 +1,33 @@
...
+
+This section shows how to prepare Akenza Core setup before creating a Webhook integration on {{% tts %}}.
+
+<!--more-->
+
+Log in to Akenza Core and navigate to the **Quick Start** tab in the **Environment** to see the steps to be taken to connect a device and manipulate the data coming from it. 
+
+{{< figure src="quick-start.png" alt="Steps to connect a device" >}}
+
+Set up a domain by selecting the **Add Domain** button in **Domains** submenu.
```
Comment (benolayinka) [inline doc/content/integrations/webhooks/cloud-integrations/akenza-core/akenza-setup/_index.md:15]:
```suggestion
Set up a domain by selecting the **Add Domain** button in the **Domains** submenu. 
```

File: doc/content/integrations/webhooks/cloud-integrations/akenza-core/akenza-setup/_index.md [hunk 17]
```diff
@@ -0,0 +1,33 @@
...
+
+<!--more-->
+
+Log in to Akenza Core and navigate to the **Quick Start** tab in the **Environment** to see the steps to be taken to connect a device and manipulate the data coming from it. 
+
+{{< figure src="quick-start.png" alt="Steps to connect a device" >}}
+
+Set up a domain by selecting the **Add Domain** button in **Domains** submenu. 
+
+Give a **Name** to your domain and choose **HTTP** for **Technology** in order to implement a Webhook integration. The **Domain Secret** is auto-generated, while **Uplink Function** and **Downlink Function** may retain a **Passthrough** value.
```
Comment (benolayinka) [inline doc/content/integrations/webhooks/cloud-integrations/akenza-core/akenza-setup/_index.md:17]:
```suggestion
Give a **Name** to your domain and choose **HTTP** for **Technology** to create a Webhook integration. The **Domain Secret** is auto-generated, while **Uplink Function** and **Downlink Function** may retain a **Passthrough** value. 
```

File: doc/content/integrations/webhooks/cloud-integrations/akenza-core/tts-setup/_index.md [hunk 18]
```diff
@@ -0,0 +1,21 @@
+---
+title: "Creating a Webhook"
+description: ""
+weight: 2
+---
+
+Once you completed the setup in Akenza Core, follow this section to create the Webhook integration.
```
Comment (benolayinka) [inline doc/content/integrations/webhooks/cloud-integrations/akenza-core/tts-setup/_index.md:7]:
```suggestion
Once you have completed the setup in Akenza Core, follow this section to create the Webhook integration.
```

File: doc/content/integrations/webhooks/cloud-integrations/akenza-core/tts-setup/_index.md [hunk 19]
```diff
@@ -0,0 +1,21 @@
...
+
+<!--more-->
+
+>Note: this section follows the [HTTP Webhooks]({{< ref "/integrations/webhooks" >}}) guide. 
+
+Fill in the **Webhook ID** field and choose **JSON** for **Webhook format**.
```
Comment (benolayinka) [inline doc/content/integrations/webhooks/cloud-integrations/akenza-core/tts-setup/_index.md:13]:
```suggestion
Fill in the **Webhook ID** field and choose **JSON** for the **Webhook format**. 
```

File: doc/content/integrations/webhooks/cloud-integrations/akenza-core/tts-setup/_index.md [hunk 20]
```diff
@@ -0,0 +1,21 @@
...
+
+<!--more-->
+
+>Note: this section follows the [HTTP Webhooks]({{< ref "/integrations/webhooks" >}}) guide. 
+
+Fill in the **Webhook ID** field and choose **JSON** for **Webhook format**. 
+
+Paste the copied **HTTP Uplink URL** from Akenza Core in the **Base URL** field.
+
+To send uplink messages to Akenza Core, check the **Enabled** box next to the **Uplink message**.
```
Comment (benolayinka) [inline doc/content/integrations/webhooks/cloud-integrations/akenza-core/tts-setup/_index.md:17]:
```suggestion
To send uplink messages to Akenza Core, check the **Enabled** box next to **Uplink message**. 
```

**Comment by nejraselimovic** [comment]
@laurensslats please approve if there's nothing left to fix

File: doc/content/integrations/webhooks/cloud-integrations/akenza-core/tts-setup/_index.md (hunk 15 above)
Comment (laurensslats) [reply doc/content/integrations/webhooks/cloud-integrations/akenza-core/tts-setup/_index.md]: Uhf! Might need to take another language course..

**Pull request merged**